libs/login/WordPressLoginFlow/src/main/java/org/wordpress/android/login/LoginUsernamePasswordFragment.java

```diff
@@ -147,6 +147,7 @@ protected void setupContent(ViewGroup rootView) {
 
         mUsernameInput = rootView.findViewById(R.id.login_username_row);
         mUsernameInput.setText(mInputUsername);
+        mUsernameInput.setHint(getString(R.string.username));
```

Comment:
This is a user-facing change, so I'm hoping we can find another way to approach it. Otherwise, we'll need to bring in some of the original devs for the login flow to discuss.